### PR TITLE
Adjust Contact Journal Day Screen Layout (DEV)

### PR DESCRIPTION
**Pull request by axelherbstreith**
This PR prevents the Floating Action Button from overlapping the list items in the contact journal day screen.

### CODE REVIEW

File: Corona-Warn-App/src/main/res/layout/contact_diary_person_list_fragment.xml [hunk 1]
```diff
@@ -18,6 +18,8 @@
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toTopOf="parent"
+            android:paddingBottom="@dimen/spacing_huge"
+            android:clipToPadding="false"
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/res/layout/contact_diary_person_list_fragment.xml:22]: Please do it for locations too? it has the same issue

Comment (axelherbstreith) [reply Corona-Warn-App/src/main/res/layout/contact_diary_person_list_fragment.xml:22]: Done 👍

**Review by mtwalli — APPROVED** [review]
Tested on S8 Android 9,

LGTM

**Review by SamuraiKek — APPROVED** [review]
Looks good. Tested on Nokia 3 (Android 9).

**Pull request merged**